batch/batch/cleanup_sidecar.py

```diff
@@ -0,0 +1,36 @@
+import logging
+import os
+import subprocess as sp
+
+from aiohttp import web
+from hailtop import gear
+
+from .stoppable_server import StoppableServer
+
+copy_output_cmd = os.environ.get('COPY_OUTPUT_CMD')
+
+gear.configure_logging()
+log = logging.getLogger('cleanup_sidecar')
+
+app = web.Application()
+routes = web.RouteTableDef()
+server = StoppableServer(app, '0.0.0.0', 5000)
+
+
+@routes.post('/')
+async def finish(request):
+    if copy_output_cmd is not None:
+        log.info(f'copying out data')
+        try:
+            pod_log = sp.check_output(copy_output_cmd, shell=True, stderr=sp.STDOUT)
+            log.info(pod_log.decode('ascii'))
```

Comment:
Do we want this here?

Reply:
yeah, poorly named, I changed to `copy_output`. I wan the output of the copy commands to end up in stdout of the cleanup container.